Makefile

```diff
@@ -16,7 +16,7 @@ NATIVE_ARCH=$(shell uname | tr A-Z a-z)
 GOARCH=amd64
 export GOARCH
 CI_OSES=linux darwin windows
-CI_BUILD_TARGETS=$(foreach os,$(CI_OSES),$(BIN_DIR)/$(os)/kubectl-buildkit $(BIN_DIR)/$(os)/kubectl-build)
+CI_BUILD_TARGETS=$(foreach os,$(CI_OSES),$(BIN_DIR)/$(os)/kubectl-buildkit$(if $(filter windows,$(os)),.exe ) $(BIN_DIR)/$(os)/kubectl-build$(if $(filter windows,$(os)),.exe ))
```

Comment:
Looks fine, but this would have been a tad easier to read if you broke the for loop out onto a couple of lines. Something like:

CI_BUILD_TARGETS=
for os in $(CI_OSES); do \
   ifeq (os, "windows") \
     CI_BUILD_TARGETS += $(BIN_DIR)/windows/kubectl-buildkit.exe $(BIN_DIR)/windows/kubectl-build.exe \
   else \
     CI_BUILD_TARGETS += $(BIN_DIR)/$(os)/kubectl-buildkit $(BIN_DIR)/$(os)/kubectl-build \
   endif \
done